x-pack/plugins/cases/public/components/all_cases/use_cases_columns.tsx

```diff
@@ -144,17 +126,21 @@ export const useCasesColumns = ({
         render: (tags: CaseUI['tags']) => {
           if (tags != null && tags.length > 0) {
             const clampedBadges = (
-              <LineClampedEuiBadgeGroup data-test-subj="case-table-column-tags">
+              <EuiBadgeGroup data-test-subj="case-table-column-tags">
                 {tags.map((tag: string, i: number) => (
-                  <StyledEuiBadge
+                  <EuiBadge
+                    css={css`
+                      max-width: 100px;
+                      margin-right: 5px;
```

Comment:
nit: a similar spacing between elements can also be obtained with `gutterSize="s"` on `EuiBadgeGroup` 🙂